cmake/configs/default.cmake

```diff
@@ -379,7 +379,7 @@ hunter_default_version(ogles_gpgpu VERSION 0.3.6)
 hunter_default_version(oniguruma VERSION 6.8.1-p0)
 hunter_default_version(onmt VERSION 0.4.1-p2)
 hunter_default_version(openddlparser VERSION 0.1.0-p2)
-hunter_default_version(opentracing-cpp VERSION 1.6.0)
+hunter_default_version(opentracing-cpp VERSION 1.5.1)
```

Comment:
I don't think we should roll this back for everyone. We should add the `1.5.1` version and then you can specify that in the `HunterConfig.cmake` file for `Jaeger-client-cpp`. As such, can you change this back to `1.6.0`?